Stratis.Bitcoin/Features/MemoryPool/MempoolBehavior.cs

```diff
@@ -13,30 +13,86 @@
 
 namespace Stratis.Bitcoin.Features.MemoryPool
 {
+    /// <summary>
+    /// Node behavior for memory pool.
+    /// </summary>
     public class MempoolBehavior : NodeBehavior
     {
-        // Average delay between trickled inventory transmissions in seconds.
-        //  Blocks and whitelisted receivers bypass this, outbound peers get half this delay. 
+        #region Fields
+
+        /// <summary>
+        /// Average delay between trickled inventory transmissions in seconds.
+        /// Blocks and whitelisted receivers bypass this, outbound peers get half this delay. 
+        /// </summary>
         const int INVENTORY_BROADCAST_INTERVAL = 5;
-        // Maximum number of inventory items to send per transmission.
-        //  Limits the impact of low-fee transaction floods. 
+
+        /// <summary>
+        /// Maximum number of inventory items to send per transmission.
+        /// Limits the impact of low-fee transaction floods.
+        /// </summary>
         const int INVENTORY_BROADCAST_MAX = 7*INVENTORY_BROADCAST_INTERVAL;
 
+        /// <summary>
+        /// Memory pool validator injected dependency.
+        /// </summary>
         private readonly IMempoolValidator validator;
```

Comment:
This is just my opinion, no requirement. I tend to write these one line comments of properties as one liners including the summary tag:

```
<summary>Memory pool validator injected dependency.</summary>
```

I only expand to separated lines for the tags if the comment inside is multiline. But I use it only for fields and properties.

So this is just an idea to consider. No need for change if you prefer this multiline format.